CiarnuroRT/webapp/ciarnurort.js

```diff
@@ -134,6 +135,16 @@ function initSoundSystem(){
     }
     soundSystemStarted=true;
 }
+function soundFxToggle(){
+    soundFxOn=!soundFxOn;
+    if(soundFxOn){
```

Comment:
You can just use `I("initialSoundToggle").checked=I("endroundSoundToggle").checked=soundFxOn;`